SpectatorView/Calibration/Calibration/Main.cpp

```diff
@@ -16,6 +21,25 @@ LRESULT CALLBACK WndProc(HWND, UINT, WPARAM, LPARAM);
 // Entry point
 int WINAPI wWinMain(_In_ HINSTANCE hInstance, _In_opt_ HINSTANCE hPrevInstance, _In_ LPWSTR lpCmdLine, _In_ int nCmdShow)
 {
+
+	auto const filePath = CameraConfigurationFile::getMyDocumentPath() + "\\camera.cfg"s;
```

Comment:
Where is the config file read for the UnityCompositorDLL?

Reply:
Oh, I see what you mean, it's on the plugin load callback, before everything else is done

Reply:
```cpp
extern "C" void UNITY_INTERFACE_EXPORT UNITY_INTERFACE_API UnityPluginLoad(IUnityInterfaces* unityInterfaces)

	try
	{
		const auto configFilePath = CameraConfigurationFile::getMyDocumentPath() + "\\camera.cfg";
		CameraConfigurationFile configFile(configFilePath);
		configFile.readConfig();

		FrameProviderStaticConfig::width = int(configFile.getWidth());
		FrameProviderStaticConfig::height = int(configFile.getHeight());
		FrameProviderStaticConfig::fps = float(configFile.getFrameRate());
	}
	catch(std::exception& e)
	{
		(void)e;
		//Will just use the hardcoded values

		FrameProviderStaticConfig::width = FRAME_HEIGHT;
		FrameProviderStaticConfig::height = FRAME_WIDTH;
		FrameProviderStaticConfig::fps = VIDEO_FPS;
	}
```

in UnityCompositorInterface.cpp, line 568